storage/src/main/java/tech/pegasys/teku/storage/protoarray/ForkChoiceStrategy.java

```diff
@@ -466,6 +478,7 @@ public void applyUpdate(
                       block.getBlock().getParentRoot(),
                       block.getBlock().getStateRoot(),
                       block.getBlockCheckpoints(),
+                      block.getExecutionBlockNumber().orElse(UInt64.ZERO),
```

Comment:
We duplicate defaults for block number and hash here and there. Maybe put it to constants in ProtoNode? Not sure it could survive in our module dependencies

Reply:
It's not default per se though. It's more like ZERO only before merge and at terminal block. So hard to think of a name for that constant

Reply:
I added constants. Think it's better. What do you think?